### ISSUE TITLE
'mostly' kwarg not working on expect_compound_columns_to_be_unique

### ISSUE THREAD

**Issue by damnknife** [fivetran/great_expectations#5388]
I can't use the mostly kwarg on expect_compound_columns_to_be_unique expectation, the status returns as failed even when the fail ratio is below the set mostly argument. I did some checks and found out the MulticolumnMapExpectation class doesn't accept a 'mostly' argument. Is this the expected behavior? Is there a reason for this?

**Comment by AFineDayFor** [comment]
Howdy @damnknife :wave: thanks for reaching out and raising this with us :bow:


Can you share your configuration and/or workflow that is invoking [expect_compound_columns_to_be_unique](https://greatexpectations.io/expectations/expect_compound_columns_to_be_unique#examples) so we can review this further :microscope:

**Comment by damnknife** [comment]
I never fully implemented it, it returned a success while I was testing on the new suite jupyter notebook and since I was using a SQL connector I just selected a concat with the columns I wanted to use as a compound key, and it worked just fine.
I tested it with an text file and It returns False even in the mostly range:

Expectation suite:

```{
  "meta": {
    "citations": [
      {
        "batch_request": {
          "data_asset_name": "test.csv",
          "data_connector_name": "default_inferred_data_connector_name",
          "datasource_name": "test_file_datasource",
          "limit": 1000
        },
        "citation_date": "2022-06-28T16:04:04.493292Z",
        "comment": "Created suite added via CLI"
      }
    ],
    "great_expectations_version": "0.15.11"
  },
  "expectations": [
    {
      "meta": {},
      "kwargs": {
        "mostly": 0.8,
        "column_list": [
          "Order ID",
          "Order Number"
        ]
      },
      "expectation_type": "expect_compound_columns_to_be_unique"
    }
  ],
  "expectation_suite_name": "new_exp",
  "data_asset_type": null,
  "ge_cloud_id": null
}
```

I get a False in the json's 'success' key in the validator test, even with 5% fail rate, then it saves but when i try to build the data doc it returns `NameError: name 'params_with_json_schema' is not defined`

**Comment by AFineDayFor** [comment]
Howdy @damnknife :wave:, thanks for the update :bow:

Can you share the full trace if it did indeed produce one?

**Comment by billdirks** [comment]
The work for supporting `mostly` in multicolumn map expectations has been merged and will be available in the next release, v0.5.15, coming this week.
